jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    setupFilesAfterEnv: ['<rootDir>/setupTests.js', '<rootDir>/node_modules/jest-enzyme/lib/index.js'],
+    preset: '@shelf/jest-mongodb',
+    watchPathIgnorePatterns: ['<rootDir>/tmp/', '<rootDir>/node_modules/'],
+    roots: ['app'],
+    testMatch: ['**/app/api/__tests__/*.js'],
```

Comment:
I'm not sure of the testMatch syntax for jest, but this limits us to only having tests in the app/api directory.  Can we make it something like **/app/**/__tests__/*.js" so it can catch any __tests__ directory under any directory under app?